0015-ilp-addresses/0015-ilp-addresses.md

```diff
@@ -0,0 +1,158 @@
+
+- [Neighborhoods](#neighborhoods)
+- [Payment rail](#payment-rail)
+- [Ledger prefix](#ledger-prefix)
+- [Subledger and account group](#subledger-and-account-group)
+- [Account identifier](#account-identifier)
+- [Interactions](#interactions)
+
+Not all addresses contain all this information, and some addresses may use multiple segments to represent some of this information.
+
+### Neighborhoods
+
+Neighborhoods have no specific meaning, but serve as a quick shortcut for routing to the right area.
+
+- For regional currencies, use a continent code from the following table:
+    | Segment | Continent     |
+    |:--------|:--------------|
+    | `af`    | Africa        |
+    | `as`    | Asia          |
+    | `eu`    | Europe        |
+    | `na`    | North America |
+    | `sa`    | South America |
+    | `oc`    | Oceania       |
+    | `an`    | Antarctica    |
+- For decentralized crypto-currency ledgers, use the `crypto` neighborhood.
+- For naturally-ocurring commodities or other globally-distributed currencies, use the `earth` neighborhood.
+- For currencies strongly associated with a particular international corporation, use the continent of that corporation's headquarters.
+
+It may also be useful to have 1-3 levels of "sub-neighborhoods" to group together related connectors or ledgers. We have no specific recommendations at this time.
+
+### Payment Rail
+
+The payment rail refers to a specific way some accounts or ledgers are connected outside of ILP, such as ACH in the U.S., SEPA in Europe, or other online payment systems such as PayPal or Alipay. We recommend using an all-lowercase abbreviation for the payment rail if one is applicable to the address. Examples:
+
+| Segment | Payment Rail                                                       |
+|:--------|:-------------------------------------------------------------------|
+| `ach`   | [United States Automated Clearing House](https://en.wikipedia.org/wiki/Automated_Clearing_House) |
+| `sepa`  | [Single Euro Payments Area](https://en.wikipedia.org/wiki/Single_Euro_Payments_Area) |
+
+***TODO: more rails?***
+
+### Ledger Prefix
+
+The ledger prefix is the unique name of the ledger. When doing a local delivery, the connector typically uses this segment to figure out which ledger plugin to use. As a result, it's difficult (but possible) to make a connector between two ledgers that use the same ledger prefix.
+
+### Subledger
+
+Zero or more segments representing optional divisions within a ledger; for example, if a ledger is composed of partitions that run on different hardware, the partitions could be named here. If there are no fees involved in transferring between a ledger and a subledger, this can still be a "local delivery" from the connector's perspective.
```

Comment:
This local delivery concept isn't totally clear, so it would be worth adding an explanation at the top or linking to somewhere else.

It's also worth mentioning that subledgers can be used for accounting purposes, for example denoting different divisions and subdivisions within a company. They don't necessarily need to be run on different machines.